src/Resource/Otp.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace SMSGlobal\Resource;
+
+use Cassandra\Date;
+use SMSGlobal\Exceptions\AuthenticationException;
+use SMSGlobal\Exceptions\InvalidPayloadException;
+use SMSGlobal\Exceptions\InvalidResponseException;
+use SMSGlobal\Exceptions\ResourceNotFoundException;
+
+/**
+ * Class Otp
+ * @package SMSGlobal\Resource
+ */
+class Otp extends Base
+{
+    /**
+     * @var string
+     */
+    protected $resourceUri = '/otp';
+
+    /**
+     * @param string $to
+     * @param string $text
+     * @param string|null $from
+     * @param int $codeExpirt
+     * @param int $length
+     * @param \DateTime|null $messageExpiryDateTime
+     *
+     * @return array
+     * @throws AuthenticationException
+     * @throws GuzzleException
+     * @throws InvalidPayloadException
+     * @throws InvalidResponseException
+     * @throws ResourceNotFoundException
+     */
+    public function send(string $to, string $text, string $from = null, string $codeExpiry = null, string $length = null, \DateTime $messageExpiryDateTime = null): array
+    {
+        $origin = !empty($from) ? $from : '';
+        $codeExpiry = !empty($codeExpiry) ? $codeExpiry : '';
+        $length = !empty($length) ? $length : '';
+        $messageExpiryDateTime = !empty($messageExpiryDateTime) ? $messageExpiryDateTime->format(self::DATE_FORMAT) : '';
+
+        return $this->rawPayload([
+            "destination" => $to,
+            "message" => $text,
+            "origin" => $origin,
+            "codeExpiry" => $codeExpiry,
+            "length" => $length,
+            "messageExpiryDateTime" => $messageExpiryDateTime,
+        ]);
+    }
+
+    /**
+     * @param string $requestId
+     * @param string $code
+     *
+     * @return bool|array Returns true if OTP input code is verified successfully otherwise error message
+     * @throws AuthenticationException
+     * @throws InvalidPayloadException
+     * @throws InvalidResponseException
+     * @throws ResourceNotFoundException
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function verfiyById(string $requestId, string $code)
+    {
+        $origin = !empty($from) ? $from : '';
+
+        $uri = $this->prepareApiUri($this->resourceUri . '/' . $requestId);
+
+        $jsonPayload = json_encode([
+            'code' => $code,
+        ], JSON_FORCE_OBJECT);
```

Comment:
$jsonPayload = json_encode(compact('code'), JSON_FORCE_OBJECT);

Reply:
I like it